assets/src/styles/editorFonts.scss

```diff
@@ -0,0 +1,42 @@
+@font-face {
```

Comment:
This includes the contents of [this file](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/commit/1158181f2e7845c4e12403c2beb1b97e56ab8060#diff-6a303c3d9ced23f325fae852cca1af4dc97cf4c2166cb9d3aa97f5fb6e1c1190) and the next one.